geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Conversions.scala

```diff
@@ -191,6 +193,9 @@ object RichSimpleFeatureType {
   val DEFAULT_DATE_KEY    = "geomesa.index.dtg"
   val ST_INDEX_SCHEMA_KEY = "geomesa.index.st.schema"
   val USER_DATA_PREFIX    = "geomesa.user-data.prefix"
+  val KEYWORDS_KEY        = "geomesa.keywords"
+
+  val KEYWORDS_DELIMITTER = """\|"""
```

Comment:
This is actually the regex to split a (Java) String.

As an interesting note...  String.split() takes a regex pattern.  https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#split-java.lang.String-

Scala Stringlike adds a method which takes a Char:
http://www.scala-lang.org/api/2.11.5/index.html#scala.collection.immutable.StringLike@split(separator:Char):Array[String]

We could do...
val s = "foo|bar"
s.split('|')

But we'd still have to call mkString with a string...
Seq("bar", "baz").mkString("|").

Anyhow, might be worth having a separate regex and joiner defined.  Either that or @elahrvivaz  and @jahhulbert-ccri  will have a better idea;).